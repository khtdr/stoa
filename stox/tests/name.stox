#!/usr/bin/env stox

var name = "Joey";
print "Hello, " + name + "!";
