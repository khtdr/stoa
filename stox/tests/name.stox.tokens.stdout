VAR[3,1]
IDENTIFIER[3,5]
EQUAL[3,10]
STRING[3,12]
SEMICOLON[3,18]
PRINT[4,1]
STRING[4,7]
PLUS[4,17]
IDENTIFIER[4,19]
PLUS[4,24]
STRING[4,26]
SEMICOLON[4,29]
